cluster-scope/README.md

```diff
@@ -34,6 +37,11 @@ The directories and files in this structure fulfill the following purpose:
 - `kustomization.yaml` : Kustomization file including a single resource, which is a sibling file within the folder. Can pull in `components` for additional bootstrapping.
 - `<resource_kind_singular>.yaml` : **Single** resource manifest with a `.spec` that can be deployed to all the clusters.
 
+#### bundles/
+Bundled manifests are manifests that are dependent on each other and should be deployed together on the same cluster. You should create a bundle that references all these manifests from a single kustomization.yaml file following the structure above. This bundle can then be easily imported to the target cluster's by adding it to overlays/<env>/<cluster_name>/kustomization.yaml. Bundle related resources by including ( or removing) a single line in the relevant overlay.
```

Comment:
can you add code formatting to: overlays/<env>/<cluster_name>/kustomization.yaml 
e.g. `overlays/<env>/<cluster_name>/kustomization.yaml` 

`( or removing)` change to `(or removing)`

Reply:
ok! will do it right away

Reply:
done!